packages/babel-plugin/src/styled/__tests__/behaviour.test.tsx

```diff
@@ -254,4 +254,38 @@ describe('styled component behaviour', () => {
     expect(actual).toInclude('({as:C="div",style,textSize,...props},ref)');
     expect(actual).toInclude('"--_1j0t240":"super"+((()=>{return textSize;})()||"")+"big"');
   });
+
+  it('should collect args as styles', () => {
+    const actual = transform(`
+      import { styled } from '@compiled/react';
+
+      const ListItem = styled.div(
+        { color: 'darkorchid' },
+        { fontSize: 12 },
+      );
+    `);
+
+    expect(actual).toIncludeMultiple([
+      '{color:darkorchid}',
+      '{font-size:12px}',
+      'ax(["_syaz1paq _1wyb1fwx",props.className])',
+    ]);
+  });
+
+  it('should add missing semicolon', () => {
```

Comment:
Looks like test statement doesn't seems to match what we are expecting 🤔. Did we miss something? 🤔

```ts
expect(actual).toIncludeMultiple([
  '{color:red}',
  '{font-size:20px}',
  'ax(["_syaz5scu _1wybgktf",props.className])',
]);
```

Reply:
Ah it's because it gets into atomic CSS so it's a bit cryptic. You can see the string is missing a semi colon - this test makes sure that scenario still works. 

We could change to a should not throw? 🤔

Reply:
hehe. Test is right. What I was saying that test statement and expectation are not matching. Semicolon is not present in expectation.

Reply:
Hehe ok, maybe can just call it "should not throw when template literal CSS has not terminating semi colon"

Reply:
Cool. And expect should not throw error. Otherwise it should if we remove the fix.